rfcs/README.md

```diff
@@ -0,0 +1,125 @@
+# Carbon RFCs
+
+Many changes, including bug fixes and documentation improvements, can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put
+through a bit of a design process and produce a consensus among the Carbon
+core team.
+
+The "RFC" (request for comments) process is intended to provide a
+consistent and controlled path for new features to enter the project.
+
+[Active RFC List](https://github.com/Sage/carbon/pulls?q=is%3Apr+is%3Aopen+label%3A%22RFC%22)
+
+Carbon is still **actively developing** this process, and it will still change as
+more features are implemented and the community settles on specific approaches
+to feature development.
+
+## When to follow this process
+
+You should consider using this process if you intend to make "substantial"
+changes to Carbon or its documentation. Some examples that would benefit
+from an RFC are:
+
+   - A new component
+   - The removal of a component that has been shipped
+   - Breaking changes to the interface/props of a component
+   - The addition of runtime exceptions
+   - Changes to the way Carbon is imported
+   - Changes to the way Carbon is built
+
+The RFC process is a great opportunity to get more eyeballs on your proposal
+before it becomes a part of a released version of Carbon. Quite often, even
+proposals that seem "obvious" can be significantly improved once a wider
+group of interested people have a chance to weigh in.
+
+The RFC process can also be helpful to encourage discussions about a proposed
+feature as it is being designed, and incorporate important constraints into
+the design while it's easier to change, before the design has been fully
+implemented.
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganising or refactoring Carbon internals
+  - Addition or removal of warnings
+  - Additions or modifications that don't change the external Carbon interface
+  - Additions only likely to be _noticed by_ other implementors-of-Carbon,
+  invisible to users-of-Carbon.
+
+## What the process is
+
+In short, to get a major feature added to Carbon, one usually first gets
+the RFC merged into the repo as a markdown file. At that point the RFC
+is 'active' and may be implemented with the goal of eventual inclusion
+into Carbon.
+
+* Fork the repo – http://github.com/sage/carbon
+* * If you have access, you may clone the repo and submit a PR from a branch.
+* Copy `rfcs/template.md` to `rfcs/text/my-feature.md` (where 'my-feature' is descriptive).
+* Fill in the RFC. Put care into the details: **RFCs that do not
+present convincing motivation, demonstrate understanding of the
```

Comment:
Thinking about the structure of this sentence, it could be parsed like this:

> RFCs that
> 1. do not present convincing motivation,
> 2. demonstrate understanding of the impact of the design, or
> 3. are disingenuous about the drawbacks or alternatives,
>
> tend to be poorly-received.

where item :two: has the opposite meaning of what's intended!

Or the sentence could be parsed like this:

> RFCs that do not
> 1. present convincing motivation,
> 2. demonstrate understanding of the impact of the design, or
> 3. are disingenuous about the drawbacks or alternatives,
>
> tend to be poorly-received.

but then item :three: doesn't fit with the initial phrase (_"RFCs that do not: are disingenuous about the drawbacks or alternatives"_).

I think this change would make things clearer:

```suggestion
present convincing motivation, do not demonstrate understanding of the
```

so that the sentence is parsed as:

> RFCs that
> 1. do not present convincing motivation,
> 2. do not demonstrate understanding of the impact of the design, or
> 3. are disingenuous about the drawbacks or alternatives,
>
> tend to be poorly-received.

Reply:
Do you think it would read better as `do not present convincing motivation, or`?

Reply:
I think that would still have the potential for misunderstanding:

> RFCs that _**do not present convincing motivation, or**_ demonstrate understanding of the impact of the design, or are disingenuous about the drawbacks or alternatives, tend to be poorly-received.

That could be parsed as:

> RFCs that
> 1. _**do not present convincing motivation, or**_
> 2. demonstrate understanding of the impact of the design, or
> 3. are disingenuous about the drawbacks or alternatives,
>
> tend to be poorly-received.

where the meaning of :two: is still the opposite of what's intended.

I think we still need to add the words `do not` before `demonstrate understanding of the impact of the design`.